### DISCUSSION TITLE
An option to decrypt file name(s) without having the file(s) locally present

### DISCUSSION THREAD

**Discussion by kzshantonu** [samuel-lucas6/Kryptor#63] [Ideas]
Could be handy to decrypt file names of files stored on cloud storage, to decide which file I need to download for example.

**Comment by kzshantonu** [comment]
Two-way is even better. Plaintext to ciphertext and ciphertext to plaintext (the way rclone does it)

**Comment by samuel-lucas6** [comment]
Whilst I agree that would be handy, that is not feasible for this type of project. The file name is also not revealed until after decryption has finished in case someone has modified the file but not the header.
